ops/services/virtual_network/_var.tf

```diff
@@ -0,0 +1,17 @@
+variable "project" {
+  default = "prime"
+}
+
+variable "name" {
+  # should be app_name
```

Comment:
Could probably change this now with minimal hassle, and probably should—I started with a very copy-paste-friendly take on this in order to avoid unintentional changes.

Reply:
What would you change about this? A variable with a default value seems like a reasonable way to handle it.

Reply:
The variable `name` is not very helpful as to what it's supposed to hold—I changed it to `app_name` to reflect that. A lot of the code in here was parameterized as if we expected multiple PRIME apps to use it, and we don't currently expect that to happen, so this is arguably something we should delete instead of porting, but I was trying to keep things relatively stable.

Reply:
I realize the issue here is actually that I didn't delete the comment when I fixed the thing I was commenting on. 😆